server.js

```diff
@@ -535,8 +553,6 @@ Game.prototype.finish = function(reason) {
                        })
                      };
     game.endMessage = endMessage;
-
-    db.snapshot();
```

Comment:
Please add an empty `snapshot()` function to `redis.js` so that the functionality is retained for `node-dirty`.